src/main/java/spoon/support/reflect/code/CtJavaDocImpl.java

```diff
@@ -96,6 +99,75 @@ public String getShortDescription() {
 		}
 	}
 
+	/**
+	 * Parses the content string to split in two: the description and the Javadoc tags
+	 */
+	@Override
+	public <E extends CtComment> E setContent(String content) {
+		tags.clear();
+
+		// avoiding NPE later
+		if (content == null) {
+			content = "";
+		}
+
+		String longDescription = "";
+		String currentTagContent = "";
+		CtJavaDocTag.TagType currentTag = null;
+
+		String[] lines = cleanComment(content).split("\n|\r\n|\r");
```

Comment:
cleanComment always returns string with CtComment.LINE_SEPARATOR as line separator.
So it is enough to write `split("\n")

Reply:
fixed